frontend/src/components/ShoppingListDetail.tsx

```diff
@@ -8,6 +8,7 @@ import {ShoppingList} from "./ShoppingListSchema.ts";
 export default function ShoppingListDetails() {
     const { id } = useParams<{ id: string }>();
     const [list, setList] = useState<ShoppingList | null>(null);
+    const [showCompleted, setShowCompleted] = useState(false);
```

Comment:
Bei Boolean Werten verwendet man in Regel "is" am Anfang, hier wäre es "isCompleted"